content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Join the DVC Ambassador Program!'
+date: 2020-05-08
+description: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+descriptionLong: |
+  We're launching our ambassador program for people all around the world to get involved in 
+  the DVC community.
+picture: 2020-05-08/Ambassador_Header.png
+author: marcel_rd
+commentsUrl: https://discuss.dvc.org/t/new-video-reimagining-devops-for-machine-learning/341
+tags:
+  - Ambassador
+  - Volunteer
+  - Talks
+  - Meetup
+  - Outreach
+---
+
+Software can be everywhere, but developers can’t - that’s why ambassadors, folks
+who do outreach and community building around projects they love, are a key part
+of the open-source community. DVC is starting an ambassador program to help
+people who are passionate about our mission get involved.
+
+What is that mission, exactly? DVC is founded on the principle of engineering
+solutions for making data science and machine learning rigorous and
+reproducible. If this matters to you, too, you might be a good fit for our
+ambassador program!
+
+As an ambassador, you’ll act as a bridge between DVC in your community. There
+are lots of ways to do this, big and small. For example:
+
+- Write a blog post talking about how you use DVC in your projects
+- What about creating a network of DVC users and data scientists in your town?
+  Even though we’re self-isolating now, you can still organize online meetups.
+  [We already did one!](https://tulu.la/events/dvc-virtual-meetup-2020-00032c).
+  We help cover costs to organize meetups.
+- Do you want to talk about DVC at your office, or at a conference? We help
+  speakers develop talks, and we have some discretionary funds for travel on a
+  case-by-case basis.
+- Want to develop a feature for DVC? We welcome contributions to the code base,
+  even if it’s your first pull request ever.
+
+Being an ambassador means getting closer to the team in charge of DVC, but at
+the same time, it means going farther to reach people outside the organization-
+including people who don’t know about DVC yet, people who need some help getting
+started, and people who are already excited about our mission and want to find
+meaningful ways to pitch in.
+
+It’s challenging, no doubt, but being an ambassador can be an amazing
+experience! Before becoming DVC’s first official ambassador, I served as a
```

Comment:
It was a bit not expected to see your personal voice here in the middle of the post. Does it make sense to make your intro in the beginning? In the 1st or 2nd paragraph?

Reply:
I don't think this really fits in the introduction, it's more like one quick comment to emphasize how cool it is to be an ambassador, which is the goal of this section. It's fine if you think it's better to make it less personal, and remove it, or maybe move it around. This was actually a suggestion by @andronovhopf.

Reply:
i moved it into the 2nd paragraph